packages/helper/src/slack/index.ts

```diff
@@ -88,6 +88,7 @@ async function sendSlackFeedbackMessage(slackEmail: ISlackFeedbackMsg): Promise<
 
 async function sendDonationNotificationToSlack(donationInfo: IDonationSlack): Promise<boolean> {
 	const { service, userDonation, donor, wishCard, amount } = donationInfo;
+	const userContribution = userDonation || 0.0;
```

Comment:
why 2 variable names?

Reply:
Ah, you are right, lol; not sure why I did it like that xD Will fix it :+1: